google/cloud/forseti/services/utils.py

```diff
@@ -169,7 +169,7 @@ def to_full_resource_name(full_parent_name, resource_type_name):
         str: full_resource_name of the child
     """
 
-    return '{}{}/'.format(full_parent_name, resource_type_name)
+    return '{}/{}'.format(full_parent_name, resource_type_name)
```

Comment:
Hi @jceresini , all full name ends with a '/' that's why in the code we have '{parent_full_name}{resource_type_name}/', is there any reason why you want to switch the position?

Reply:
Please take a look at my comments above. I really dont think that trailing slash is intentional, theres nothing separating the parent from the resource. And as I mentioned, this could lead to ambiguous or even overlapping full_name values. 

In the schema currently:
parent_type_name = `[parent_type]`/`[parent_name]`
type_name = `[type]`/`[name]`
full_name = `[parent_type_name][type_name]`/

Note that the last one listed there has no field separator between the 2 values its comprised of.

Reply:
Oh, sorry; my comments are in the issue linked above, not in this thread. #1703

Reply:
Hi @jceresini , you would be right if we are passing in parent_type_name into this method, but the method is taking in parent_full_name and parent_full_name is a full name so it should end with '/', can you look at my comment in issue #1703 ? We are curious about the environment you installed Forseti rc3 on.